change/@fluentui-make-styles-c09cef4a-261d-43d0-8f3b-47c6314af62c.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Change makeStyle to makeStyleCompat",
```

```suggestion
  "comment": "Change makeStyles to makeStyleCompat",
```

nit, the same is for other change files